searcch_backend/models/model.py

```diff
@@ -801,3 +801,35 @@ class ImporterSchedule(db.Model):
     def __repr__(self):
         return "<ImporterSchedule(id=%r,artifact_import=%r,importer_instance=%r,schedule_time=%r" % (
             self.id, self.artifact_import, self.importer_instance, self.schedule_time)
+
+
+# Models to capture Statistical Data
+
+class StatsArtifactViews(db.Model):
+    __tablename__ = "stats_views"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    artifact_id = db.Column(db.Integer, db.ForeignKey("artifacts.id"), nullable=False)
```

Comment:
This table needs to be added to the list of tables whose entries are purged when artifacts are deleted; see searcch_backend/api/resources/artifact.py .

Reply:
Sure David. I'll look into it.